lib/GADS/Audit.pm

```diff
@@ -167,6 +167,16 @@ sub logs
         schema     => $self->schema,
         init_value => $_->{user},
     )->presentation(type => 'person', site => $site) foreach @logs;
+    
+    my $df = DateTime::Format::Strptime->new(
```

Comment:
I don't like this, but it appears the datetime is pulled from the DB as string, so it's the only way I could do it

Reply:
Ah, you're getting bitten by the `$rs->result_class('DBIx::Class::ResultClass::HashRefInflator')` line above. If you remove that they will be returned as `DateTime` objects. You will then need to adjust the other uses of the result objects accordingly (such as the user, I'm not sure how on initial glance)